pytorch_lightning/strategies/ddp.py

```diff
@@ -95,7 +98,9 @@ def __init__(
         self._ddp_comm_state = ddp_comm_state
         self._ddp_comm_hook = ddp_comm_hook
         self._ddp_comm_wrapper = ddp_comm_wrapper
-        self._model_averaging_period = model_averaging_period
+        if _TORCH_GREATER_EQUAL_1_10:
+            self._model_averaging_period = model_averaging_period
```

Comment:
also, what is the attribute when PL<1.10

Reply:
do you mean PT? I will remove the `if _TORCH_GREATER_EQUAL_1_10:`